packages/react-native/src/Authenticator/Defaults/ResetPassword/ResetPassword.tsx

```diff
@@ -16,8 +16,8 @@ const BACK_TO_SIGN_IN = 'Back to Sign In';
 
 const ResetPassword: ResetPasswordComponent = ({
   error,
-  Footer,
-  Header,
+  Footer = ResetPassword.Footer,
+  Header = ResetPassword.Header,
```

Comment:
These are passed in via `useAuthenticatorRoute` and should not have default values